lib/destination-tile/styles.css

```diff
@@ -140,6 +140,7 @@
 
 .videoIconContainer {
   display: inline-block;
+  z-index: 1000000;
```

Comment:
you sure a *million* is enough...? 😜 
![one million](http://jeffrose.wpengine.netdna-cdn.com/wp-content/uploads/2011/12/dr.-evil-million-dollar-term-policy-300x241.jpg)